packages/jest-cli/src/TestRunner.js

```diff
@@ -30,6 +30,7 @@ const promisify = require('./lib/promisify');
 const runTest = require('./runTest');
 const snapshot = require('jest-snapshot');
 const workerFarm = require('worker-farm');
+const WatchState = require('./jest').WatchState;
```

Comment:
hm, this circular dependency is not great. Can we put `WatchState` into a separate file next to `TestRunner.js`?